src/pytest_html/nextgen.py

```diff
@@ -0,0 +1,66 @@
+import json
+from typing import Any
```

Comment:
Thanks for adding typing! Can you add [mypy](https://mypy.readthedocs.io/en/stable/) to the pre commit config as part of the next gen migration so we can enforce it?

We could just follow what [pytest](https://github.com/pytest-dev/pytest/blob/master/.pre-commit-config.yaml#L54) does. I think it's the easiest way to do it.

I don't think it's necessary in this PR, but can we do it as part of next gen? It would be good to have 100% typing if possible

Reply:
I opened an issue to track typing (https://github.com/pytest-dev/pytest-html/issues/434)

I also added mypy to our linting target and began work on typing the project in https://github.com/pytest-dev/pytest-html/pull/435